.github/scripts/build-openblas.sh

```diff
@@ -4,7 +4,7 @@ source `dirname $0`/config.sh
 
 echo "::group::Build OpenBLAS"
     cd $SOURCE_PATH/$OPENBLAS_VERSION
-    make BINARY=64 CC=aarch64-w64-mingw32-gcc HOSTCC=gcc NOFORTRAN=1 TARGET=ARMV8 $BUILD_MAKE_OPTIONS
+    make BINARY=64 CC=aarch64-w64-mingw32-gcc CFLAGS=-Wno-incompatible-pointer-types HOSTCC=gcc NOFORTRAN=1 TARGET=ARMV8 $BUILD_MAKE_OPTIONS
```

Comment:
Recent update of GCC to upstream broke OpenBLAS build with `-Wincompatible-pointer-types` being set as default. Alternatively the fix on OpenBLAS side would be:

```patch
diff --git a/ctest/c_cblat1c.c b/ctest/c_cblat1c.c
index b4c512436..5ff69ad13 100644
--- a/ctest/c_cblat1c.c
+++ b/ctest/c_cblat1c.c
@@ -481,7 +481,7 @@ static real c_b43 = (float)1.;
 		stest1_(&r__1, &strue4[np1 - 1], &strue4[np1 - 1], sfac);
 	    } else if (combla_1.icase == 8) {
 /*              .. CSCAL .. */
-		cscal_(&combla_1.n, &ca, cx, &combla_1.incx);
+		cscal_(&combla_1.n, &ca.r, &cx->r, &combla_1.incx);
 		ctest_(&len, cx, &ctrue5[(np1 + combla_1.incx * 5 << 3) - 48],
 			 &ctrue5[(np1 + combla_1.incx * 5 << 3) - 48], sfac);
 	    } else if (combla_1.icase == 9) {
@@ -515,7 +515,7 @@ static real c_b43 = (float)1.;
 	    mwpcs[i__1].r = (float)1., mwpcs[i__1].i = (float)1.;
 /* L80: */
 	}
-	cscal_(&c__5, &ca, cx, &combla_1.incx);
+	cscal_(&c__5, &ca.r, &cx->r, &combla_1.incx);
 	ctest_(&c__5, cx, mwpct, mwpcs, sfac);
     } else if (combla_1.icase == 9) {
 /*        CSSCALTEST */
diff --git a/ctest/c_zblat1c.c b/ctest/c_zblat1c.c
index f7c0515fc..e5c15ace9 100644
--- a/ctest/c_zblat1c.c
+++ b/ctest/c_zblat1c.c
@@ -422,7 +422,7 @@ static doublereal c_b43 = 1.;
 		stest1_(&d__1, &strue4[np1 - 1], &strue4[np1 - 1], sfac);
 	    } else if (combla_1.icase == 8) {
 /*              .. ZSCALTEST .. */
-		zscaltest_(&combla_1.n, &ca, cx, &combla_1.incx);
+		zscaltest_(&combla_1.n, &ca.r, cx, &combla_1.incx);
 		ctest_(&len, cx, &ctrue5[(np1 + combla_1.incx * 5 << 3) - 48],
 			 &ctrue5[(np1 + combla_1.incx * 5 << 3) - 48], sfac);
 	    } else if (combla_1.icase == 9) {
@@ -456,7 +456,7 @@ static doublereal c_b43 = 1.;
 	    mwpcs[i__1].r = 1., mwpcs[i__1].i = 1.;
 /* L80: */
 	}
-	zscaltest_(&c__5, &ca, cx, &combla_1.incx);
+	zscaltest_(&c__5, &ca.r, cx, &combla_1.incx);
 	ctest_(&c__5, cx, mwpct, mwpcs, sfac);
     } else if (combla_1.icase == 9) {
 /*        ZDSCALTEST */
@@ -647,7 +647,7 @@ static doublereal c_b43 = 1.;
 			 sfac);
 	    } else if (combla_1.icase == 3) {
 /*              .. ZAXPYTEST .. */
-		zaxpytest_(&combla_1.n, &ca, cx, &combla_1.incx, cy, &
+		zaxpytest_(&combla_1.n, &ca.r, cx, &combla_1.incx, cy, &
 			combla_1.incy);
 		ctest_(&leny, cy, &ct8[(kn + (ki << 2)) * 7 - 35], &csize2[
 			ksize * 7 - 7], sfac);
diff --git a/lapack-netlib/SRC/cpstf2.c b/lapack-netlib/SRC/cpstf2.c
index f0ac9bf92..2ff4b11d1 100644
--- a/lapack-netlib/SRC/cpstf2.c
+++ b/lapack-netlib/SRC/cpstf2.c
@@ -256,7 +256,7 @@ static char junk[] = "\n@(#)LIBF77 VERSION 19990503\n";
 #define myceiling_(w) {ceil(w)}
 #define myhuge_(w) {HUGE_VAL}
 //#define mymaxloc_(w,s,e,n) {if (sizeof(*(w)) == sizeof(double)) dmaxloc_((w),*(s),*(e),n); else dmaxloc_((w),*(s),*(e),n);}
-#define mymaxloc_(w,s,e,n) dmaxloc_(w,*(s),*(e),n)
+#define mymaxloc_(w,s,e,n) smaxloc_(w,*(s),*(e),n)
 
 /* procedure parameter types for -A and -C++ */
 
diff --git a/lapack-netlib/SRC/cpstrf.c b/lapack-netlib/SRC/cpstrf.c
index e8cb87df4..0275b1ec2 100644
--- a/lapack-netlib/SRC/cpstrf.c
+++ b/lapack-netlib/SRC/cpstrf.c
@@ -256,7 +256,7 @@ static char junk[] = "\n@(#)LIBF77 VERSION 19990503\n";
 #define myceiling_(w) {ceil(w)}
 #define myhuge_(w) {HUGE_VAL}
 //#define mymaxloc_(w,s,e,n) {if (sizeof(*(w)) == sizeof(double)) dmaxloc_((w),*(s),*(e),n); else dmaxloc_((w),*(s),*(e),n);}
-#define mymaxloc_(w,s,e,n) dmaxloc_(w,*(s),*(e),n)
+#define mymaxloc_(w,s,e,n) smaxloc_(w,*(s),*(e),n)
 
 /* procedure parameter types for -A and -C++ */
 
diff --git a/lapack-netlib/SRC/spstf2.c b/lapack-netlib/SRC/spstf2.c
index f408568b9..d9624559d 100644
--- a/lapack-netlib/SRC/spstf2.c
+++ b/lapack-netlib/SRC/spstf2.c
@@ -256,7 +256,7 @@ static char junk[] = "\n@(#)LIBF77 VERSION 19990503\n";
 #define myceiling_(w) {ceil(w)}
 #define myhuge_(w) {HUGE_VAL}
 //#define mymaxloc_(w,s,e,n) {if (sizeof(*(w)) == sizeof(double)) dmaxloc_((w),*(s),*(e),n); else dmaxloc_((w),*(s),*(e),n);}
-#define mymaxloc_(w,s,e,n) dmaxloc_(w,*(s),*(e),n)
+#define mymaxloc_(w,s,e,n) smaxloc_(w,*(s),*(e),n)
 
 /* procedure parameter types for -A and -C++ */
 
diff --git a/lapack-netlib/SRC/spstrf.c b/lapack-netlib/SRC/spstrf.c
index 5d446aa95..2ae2e929b 100644
--- a/lapack-netlib/SRC/spstrf.c
+++ b/lapack-netlib/SRC/spstrf.c
@@ -256,7 +256,7 @@ static char junk[] = "\n@(#)LIBF77 VERSION 19990503\n";
 #define myceiling_(w) {ceil(w)}
 #define myhuge_(w) {HUGE_VAL}
 //#define mymaxloc_(w,s,e,n) {if (sizeof(*(w)) == sizeof(double)) dmaxloc_((w),*(s),*(e),n); else dmaxloc_((w),*(s),*(e),n);}
-#define mymaxloc_(w,s,e,n) dmaxloc_(w,*(s),*(e),n)
+#define mymaxloc_(w,s,e,n) smaxloc_(w,*(s),*(e),n)
 
 /* procedure parameter types for -A and -C++ */
```

Reply:
I was considering that but I am not sure I want to spent additional time investigating where the regression originates from, at least for now.

Reply:
if something happened at weekend with daily rebase on the second day backup will be overwritten by already broken origin. A weekly backup might help to react and not to lose data or at least minimize impact.